### PR TITLE
Add info about workarounds required on WSL

### PR DESCRIPTION
**Pull request by nathanielcwm**
Squash

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -41,6 +42,11 @@ For installation of Jekyll please follow https://jekyllrb.com/docs/installation/
 
 4. bundle install 
 
+### WSL notes
+If you're running through WSL you need to use the flag `--force-polling` or it will not regenerate automatically on changes.
```
Comment (RoboticMind) [inline README.md:46]: Probably good to specify that the `--force-polling` flag is for jekyll

File: README.md [hunk 2]
```diff
@@ -41,6 +42,11 @@ For installation of Jekyll please follow https://jekyllrb.com/docs/installation/
...
+
+If you wish to accesss it from another device you need to use [this workaround](https://github.com/microsoft/WSL/issues/4150#issuecomment-504209723). Remember to change the ports that are opened.
```
Comment (RoboticMind) [inline README.md:48]: Minor, but noticed a typo here with an extra s
```suggestion
If you wish to access it from another device you need to use [this workaround](https://github.com/microsoft/WSL/issues/4150#issuecomment-504209723). Remember to change the ports that are opened.
```

File: README.md (hunk 1 above)
Comment (nathanielcwm) [reply README.md]: I feel like moving it under the jekyll section would make it self explanatory.

Comment (RoboticMind) [reply README.md]: Somewhat, but since git is mentioned directly above in the windows section, so it's not always obvious

Comment (RoboticMind) [reply README.md]: With the change you've made, I think it's now fine though

**Review by RoboticMind — APPROVED** [review]

**Pull request merged**